data/human/news.txt

```diff
@@ -2150,3 +2150,164 @@ news "graffiti on spaceport wall"
 			"Veni, vidi, hyperjumped away."
 			"Those who dance will be called fools by those who cannot hear the music."
 			"Better half a sentence, "
+
+news "High Tension Situation"
+	location
+		attribute "ccor"
+name
+	"Standoff"
+message
+	word
+		"CCOR Guerrillas "
+		"Guerrillas in CCOR colors "
+	word
+		"are "
+	word
+		"distributing pampflets, "
+		"delivering aid, "
+		"rallying against the local authorities, "
+		"supporting local protests, "
+	word
+		"when "
+	word
+		"planetary authorities "
+		"local pirates "
+	word
+		"acost "
+		"confront "
+	word
+		"them to "
+	word
+		"dispurse. "
+		"abandon their objective. "
+		"come quietly. "
+	word
+		"The Guerrillas "
+	word
+		"bluster "
+		"saber-rattle "
+	word
+		"for a bit in response "
+	word
+		"but it all ends with them dispersing."
+		"and their opposition backs down."
+
+phrase "A Light Ship"
+	word
+		"A Sparrow"
+		"An Enforcer"
+		"A Shuttle"
+		"A Heavy Shuttle"
+		"A Star Barge"
+		"A Berserker"
+		"A Fury"
+		"A Hawk"
+		"A Wasp"
+
+phrase "Pronoun"
+	word
+		"his"
+		"hers"
+		"their"
+
+phrase "Careless Ends Well"
+	word
+		"turns around to reenter their ship, "
+	word
+		"as it's struck by a wave, nudging it off the platform. The captain then moves the ship to a lowerable pad."
+		"moving it to a lowerable pad before it could be caught by a wave."
```

Comment:
```suggestion
		" as it's struck by a wave, nudging it off the platform. The captain then moves the ship to a lowerable pad."
		", moving it to a lowerable pad before it could be caught by a wave."
```